src/app.js

```diff
@@ -1,8 +1,9 @@
 import express from 'express'
 import { Dynamo } from './dynamo.js'
-import { heartbeatInterval, JOIN_ROOM } from './constants.js'
+import { GAME_STATE, heartbeatInterval, JOIN_ROOM, MOVE_PLAYED } from './constants.js'
```

Comment:
`heartbeatInterval` should probably be `HEARTBEAT_INTERVAL` if treated as constant